common/tools/dev-tool/src/commands/admin/migrate-package.ts

```diff
@@ -372,6 +372,12 @@ function setFilesSection(packageJson: any): void {
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 function addTypeScriptHybridizer(packageJson: any, options: { browser: boolean }): void {
+  // Do not remove subpath exports for modular package's package.json
+  if (packageJson["tshy"]) {
```

Comment:
Wouldn't we want to pick up the subpaths still if tshy isn't there?

Reply:
do we still have existing cases that are using subpaths without leveraging tshy to bundle? For codegen, we have https://github.com/Azure/autorest.typescript/blob/main/packages/rlc-common/src/metadata/packageJson/packageCommon.ts#L102-L114 this to gurantee. But I am not sure for manual authoring libraries.

Reply:
Probably not at this point, no.